addons/sale_project/models/project_task.py

```diff
@@ -167,6 +167,18 @@ def write(self, vals):
             self._ensure_sale_order_linked([sol_id])
         return task
 
+    @api.model
+    def default_get(self, fields_list):
+        default = super().default_get(fields_list)
```

Comment:
```suggestion
    def default_get(self, fields):
        default = super().default_get(fields)
```

[update arguments ](https://github.com/odoo/odoo/pull/218334/files)

default_get should be defined before the field declaration. [Doc](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html?highlight=default_get#symbols-and-conventions)